mention-bot.js

```diff
@@ -415,8 +420,8 @@ async function guessOwnersForPullRequest(
 ): Promise<Array<string>> {
   var diff = await fetch(repoURL + '/pull/' + id + '.diff');
   var files = parseDiff(diff);
-  var defaultOwners = getDefaultOwners(files, config.alwaysNotifyForPaths);
-
+  var defaultOwners = getMatchingOwners(files, config.alwaysNotifyForPaths);
+  var fallbackOwners = getMatchingOwners(files, config.fallbackNotifyForPaths);
   if (!config.findPotentialReviewers) {
       return defaultOwners;
```

Comment:
@CarlRosell @vjeux should we make this `return defaultOwners || fallbackOwners;` (or something similar) instead, in order to also have  support for the fallback notify users mechanism (being introduced in this PR) when `findPotentialReviewers` is disabled?
What do you guys think?